public/js/index.js

```diff
@@ -209,6 +209,10 @@ function initInstall() {
         return;
     }
 
+    $('#db_host').val("127.0.0.1:3306");
+    $('#db_user').val("root");
```

Comment:
That's only true for MySQL/MariaDB not for PostgreSQL & MSSQL. Besides that I don't think that we should encourage users to use a root account "just because it's there".